Technical-Steering-Committee/squads.md

```diff
@@ -0,0 +1,108 @@
+
+In the case of an Incubation Stage squad that is not renewed with Zowe, the trademark will be returned to the project maintainers or an organization they designate.
+
+In order to move to the active stage, following requirements must be met and demonstrated by the squad. The squad:
+ * Have committers from at least two organizations.
+ * Follow the TSC agreed Best practices.
+ * Have a designated TSC representative. Provide updated Governance and Committers files where applicable. The current committers and contributors are in the GitHub as outlined in the TSC [Contributing guidelines](contributing.md)
+
+The squads are expected to move from the incubation stage within at most two major version releases. If agreed with TSC they can stay in the Incubation stage longer when needed. TSC will consult ZAC for guidance on whether the incubation should be longer  
+
+#### Active Stage
+
+Active stage squad represents a mature squad working within the Zowe project who is actively collaborating with the other squads. Squads can remain in the active stage indefinitely. 
+
+#### Emeritus Stage
+
+Squads have lifecycles, and often in open source the relevance for a given squad over time can diminish. Nonetheless, having a home for artifacts no longer actively being developed by any squad is crucial for long-term sustainability and asset management. 
+
+Squads only can enter the Emeritus Stage by either:
+
+* On request from the squad itself, requiring a 2/3rd votes of all active squad committers
+* By a majority vote of the TSC if there has been insufficent activity in the squad over the course of 6 months or if the squad doesn't follow the best practices despite the reminders from the TSC.
+
+When in the Emeritus Stage, the squad's code repository administration is transferred to a designated individual by the Zowe. No new features or bug fixes will be addressed, unless it is deemed a security issue. Open Mainframe Project and Zowe will hold all assets in perpetuity.
+
+For squads in the emeritus phase and the associated artifacts and code Zowe doesn't guarantee build and test environments. If someone wants to support such artifacts, it's up to them to solve this issue.
+
+A squad can move back to Active Stage following the guidelines for a project being accepted at the Active Stage above.
+
+## List of Current
+
+In this list we outline the current squads and their level of maturity. 
+
+### Proposal
+
+This list contains squads currently known to be in a proposal process. 
+
+### Incubator
+
+This list contains squads currently known to be in an incubator stage
+
+- ZEEBRA
+- Wizard Workflow
+- Zowe Mobile
+
+### Active
+
+This list contains squads currently known to be in an active stage
+
+- API Mediation Layer
+- CLI
+- Documentation
+- Explorers
```

Comment:
Maybe call out `Explorers` as `Zowe Explorers` to avoid confusion with the Zowe Desktop Explorer apps ?  Not sure of the squad's official name,  maybe Richelle or Jelly will know